spectrocloud/resource_cluster_vsphere_test.go

```diff
@@ -84,6 +84,7 @@ func prepareClusterVsphereTestData() *schema.ResourceData {
 		"count":                   1,
 		"placement":               masterPlacement,
 		"instance_type":           masterInstance,
+		"node":                    []interface{}{},
```

Comment:
Is there any assert that is actually checking the value?

Reply:
Basically node will fall under day2 operation we cant mimc d.HasChange("machine_pool")